src/classes/frModel.cls

```diff
@@ -214,10 +217,7 @@ public with sharing abstract class frModel {
             }
             record.put(field, value);
         } catch (Exception ex) {
-            insert new Error__c(Error__c = 'Field mapping exception. Object type: '+ record.getSObjectType().getDescribe().getName()
-                                +' Record Id: '+record.Id+' - Funraise Id: '+ funraiseId + ' - Field: '+field.getDescribe().getName()+' - Value: '+value
-                                +' Exception: '+ex
-                               );
+            System.debug('field: ' + ex);
```

Comment:
Can we restore the error logging here and any other places it was replaced with debugging?

Reply:
Fixed here and a couple of other places as well, let me know if i missed any and i'll get right on it ty!